alica_test_utility/src/TestContext.cpp

```diff
@@ -32,7 +32,7 @@ int TestContext::init(AlicaCreators&& creatorCtx)
         _communicator->startCommunication();
     }
 
-    if (_engine->init(std::move(creatorCtx))) {
+    if (AlicaContext::init(std::move(creatorCtx))) {
```

Comment:
this will change behaviour i.e. the engine will now be started already when this call is made, whereas earlier it was not. So, I think instead, you can remove `_initCalled` & in this method just have a single line: `AlicaContext::init(std::move(creatorCtx)), true)`